src/main/java/uk/gov/ch/controller/update/OverseasEntityManagingOfficersController.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ch.controller.update;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.ManagingOfficerCountNotFoundException;
+import uk.gov.ch.model.update.OverseasEntityManagingOfficerData;
+import uk.gov.ch.service.update.OverseasEntityManagingOfficersService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+import java.util.List;
+
+@RestController
+public class OverseasEntityManagingOfficersController {
+
+    @Autowired
+    private OverseasEntityManagingOfficersService overseasEntityManagingOfficersService;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OracleQueryApplication.APPLICATION_NAME_SPACE);
+
+    @GetMapping("/overseas-entity/{companyNumber}/managing-officers")
+    public ResponseEntity<List<OverseasEntityManagingOfficerData>> getOverseasEntityManagingOfficers(
+            @PathVariable String companyNumber) {
+
+        DataMap dataMap = new DataMap.Builder(null).companyNumber(companyNumber).build();
+
+        LOGGER.info(String.format("Calling service to retrieve managing officer data for company number %s",
```

Comment:
Did you forget about this log (line 34)?  Line 40 is good.

Reply:
fixed